src/main/java/entity/Position.java

```diff
@@ -0,0 +1,9 @@
+package entity;
+
+public enum Position {
+    CEO,
+    DepartmentHead,
```

Comment:
Should be like DEPARTMENT_HEAD, PROJECT_HEAD, MEMBER

Reply:
ok!